ocs_ci/ocs/monitoring.py

```diff
@@ -18,6 +18,252 @@
+    "ceph_rocksdb_rocksdb_write_pre_and_post_time_sum",
+    "ceph_bluefs_wal_used_bytes",
+    "ceph_pool_rd_bytes",
+    "ceph_pg_deep",
+    "ceph_rocksdb_rocksdb_write_wal_time_sum",
+    "ceph_osd_op_wip",
+    "ceph_pg_backfill_toofull",
+    "ceph_osd_flag_noup",
+    "ceph_rgw_get_initial_lat_sum",
+    "ceph_pg_scrubbing",
+    "ceph_num_objects_unfound",
+    "ceph_mon_quorum_status",
+    "ceph_paxos_lease_timeout",
+    "ceph_osd_op_r_out_bytes",
+    "ceph_paxos_begin_keys_count",
+    "ceph_bluestore_kv_sync_lat_count",
+    "ceph_osd_op_prepare_latency_count",
+    "ceph_bluefs_bytes_written_slow",
+    "ceph_rocksdb_submit_latency_sum",
+    "ceph_pg_repair",
+    "ceph_osd_op_r_latency_count",
+    "ceph_paxos_share_state_keys_sum",
+    "ceph_paxos_store_state_bytes_sum",
+    "ceph_osd_op_latency_count",
+    "ceph_paxos_commit_bytes_count",
+    "ceph_paxos_restart",
+    "ceph_rgw_get_initial_lat_count",
+    "ceph_pg_down",
+    "ceph_bluefs_slow_total_bytes",
+    "ceph_paxos_collect_timeout",
+    "ceph_pg_peered",
+    "ceph_osd_commit_latency_ms",
+    "ceph_osd_op_w_process_latency_sum",
+    "ceph_osd_weight",
+    "ceph_paxos_collect_keys_count",
+    "ceph_paxos_share_state_bytes_count",
+    "ceph_osd_op_w_prepare_latency_sum",
+    "ceph_bluestore_read_lat_sum",
+    "ceph_osd_flag_noscrub",
+    "ceph_osd_stat_bytes_used",
+    "ceph_osd_flag_norecover",
+    "ceph_pg_clean",
+    "ceph_paxos_begin",
+    "ceph_mon_election_win",
+    "ceph_osd_op_w_process_latency_count",
+    "ceph_rgw_get_b",
+    "ceph_rgw_failed_req",
+    "ceph_rocksdb_rocksdb_write_wal_time_count",
+    "ceph_rgw_keystone_token_cache_miss",
+    "ceph_disk_occupation",
+    "ceph_pg_snaptrim",
+    "ceph_paxos_store_state_keys_sum",
+    "ceph_osd_numpg_removing",
+    "ceph_pg_remapped",
+    "ceph_paxos_commit_keys_count",
+    "ceph_pg_forced_backfill",
+    "ceph_paxos_new_pn_latency_sum",
+    "ceph_osd_op_in_bytes",
+    "ceph_paxos_store_state_latency_count",
+    "ceph_paxos_refresh_latency_count",
+    "ceph_rgw_get",
+    "ceph_pg_total",
+    "ceph_osd_op_r_prepare_latency_count",
+    "ceph_rgw_cache_hit",
+    "ceph_objecter_op_w",
+    "ceph_rocksdb_submit_transaction",
+    "ceph_objecter_op_r",
+    "ceph_bluefs_num_files",
+    "ceph_osd_up",
+    "ceph_rgw_put_b",
+    "ceph_mon_election_lose",
+    "ceph_osd_op_prepare_latency_sum",
+    "ceph_bluefs_db_used_bytes",
+    "ceph_bluestore_kv_final_lat_count",
+    "ceph_pool_quota_objects",
+    "ceph_osd_flag_nodown",
+    "ceph_pg_forced_recovery",
+    "ceph_paxos_refresh_latency_sum",
+    "ceph_osd_recovery_bytes",
+    "ceph_osd_op_w",
+    "ceph_paxos_commit_bytes_sum",
+    "ceph_bluefs_log_bytes",
+    "ceph_rocksdb_submit_sync_latency_count",
+    "ceph_pool_num_bytes_recovered",
+    "ceph_pool_num_objects_recovered",
+    "ceph_pool_recovering_bytes_per_sec",
+    "ceph_pool_recovering_keys_per_sec",
+    "ceph_pool_recovering_objects_per_sec"
+]
```

Comment:
This duplicates code in https://github.com/red-hat-storage/ocs-ci/blob/19605e18c5a5cdc77ca454b01ea26343cf1ab7a4/tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py#L128